web_widget_timepicker/static/src/js/jquery.timepicker.js

```diff
@@ -0,0 +1,1225 @@
+/*!
```

Comment:
Might be better to use the CDN version of this instead of including directly. CDNJS has this library + assets avail - https://cdnjs.com/libraries/jquery-timepicker